packages/marketplace/contracts/contracts/interfaces/IMarketplace.sol

```diff
@@ -1,6 +1,9 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.28;
 
+// TODO: review this file, should we split it up into multiple files?
```

Comment:
I splitted it `IMarketplace.errors.sol` `IMarketplace.structs.sol`, `IMarketplace.events.sol` and `IMarketplace.sol`. I think looks good, it was indeed quite big. But mostly I didn't find any split, maybe the whole thing is too big... 

For example: 
https://github.com/aave/aave-v3-core/tree/master/contracts/interfaces
https://github.com/safe-global/safe-smart-account/tree/main/contracts/interfaces
https://github.com/ethereum-optimism/optimism/tree/v1.6.1/packages/contracts-bedrock/src/dispute/interfaces